### PR TITLE
Add sending support to courier, plug in sentry, log request and respo…

### PR DESCRIPTION
**Pull request by nicpottier**
…nse for all incoming/outgoing msgs

### CODE REVIEW

File: channel.go [hunk 1]
```diff
@@ -8,13 +8,28 @@ import (
 )
 
 const (
-	// ConfigAuthToken is our constant key used in channel configs for auth tokens
+	// ConfigAuthToken is a constant key for channel configs
```
Comment (rowanseymour) [inline channel.go]: Not sure how useful it is trying to make channel configs conform to any schema

**Comment by rowanseymour** [comment]
Coverage down 15% tsk tsk

**Comment by nicpottier** [comment]
Who needs tests!?!  Ya, testing is next, just wanted to cover all the use cases first.

You are talking about who calls Twilio to register the callbacks? Feels like that's a RapidPro responsibility to me but open to discussing it.

**Comment by rowanseymour** [comment]
Agree that the responsibility lies with RapidPro - see comment on https://github.com/nyaruka/rapidpro/pull/1322 for what I was thinking about activating/deactivating channels

**Pull request merged**